apollo-router/src/state_machine.rs

```diff
@@ -548,14 +554,16 @@ mod tests {
                                     (
                                         "accounts".to_string(),
                                         Subgraph {
-                                            routing_url: Url::parse("http://accounts/graphql").unwrap(),
+                                            routing_url: Url::parse("http://accounts/graphql")
+                                                .unwrap(),
                                             layers: Vec::new(),
                                         }
                                     ),
                                     (
                                         "products".to_string(),
                                         Subgraph {
-                                            routing_url: Url::parse("http://accounts/graphql").unwrap(),
+                                            routing_url: Url::parse("http://accounts/graphql")
```

Comment:
thought this was incorrect (I was expecting `http://products/graphql`) but it's not even checked. not sure `extract_routing_urls` isn't a misleading title, or if this is being checked somewhere. do you want me to investigate?

Reply:
I think this will disappear soon as routing URL is being replaced with a plugin.

Reply:
yes